lib/go-atscfg/cacheurldotconfig.go

```diff
@@ -101,11 +97,10 @@ func MakeCacheURLDotConfig(
 	// TODO verify prefix and suffix exist, and warn if they don't? Perl doesn't
 	dsName := tc.DeliveryServiceName(strings.TrimSuffix(strings.TrimPrefix(fileName, "cacheurl_"), ".config"))
 
-	ds, ok := dses[dsName]
+	_, ok := dses[dsName]
 	if !ok {
 		return text // TODO warn? Perl doesn't
 	}
-	text += ds.CacheURL + "\n"
```

Comment:
This is the only text in the file. And you removed the references to cacheurl.config and cacheurl_qstring.config. Those configs are for the plugin you're removing support for. We should consider removing this whole function, I think. And then look at what's calling it and remove up the chain.

Reply:
I believe I have properly removed up the chain, might be some I missed in atstccfg though.